etc/apparmor.d/abstractions/xchat-based

```diff
@@ -17,13 +17,15 @@
 	@{HOME}/.config/hexchat/** rwixk,
 	@{HOME}/.kde/share/config/gtkrc-2.0 r,
 	@{HOME}/.kde/share/config/oxygenrc r,
+	@{HOME}/.*/lib/python*/** r,
 
 	/bin/grep rix,
 	/bin/uname rix,
 	/bin/mkdir rix,
 	/bin/rm rix,
 
 	/dev/tty rwix,
+	/dev/null rw,
```

Comment:
Is write access really required? Did you have a denied message without write access?

Reply:
> Is write access really required?

@adrelanos  denying access to this special device file will compromise the present or future stability of the application: POSIX.1-2017 _requires_ `/dev/null` to be _always_ readable and writable by all applications and their subcomponents (modules, extensions, etc.).